packages/medusa/src/api/routes/admin/orders/process-swap-payment.ts

```diff
@@ -64,12 +68,15 @@ export default async (req, res) => {
 
   await entityManager.transaction(async (manager) => {
     await swapService.withTransaction(manager).processDifference(swap_id)
+  })
 
-    const order = await orderService.withTransaction(manager).retrieve(id, {
-      select: defaultAdminOrdersFields,
-      relations: defaultAdminOrdersRelations,
-    })
-
-    res.json({ order })
+  const order = await orderService.retrieve(id, {
```

Comment:
question: No totals?

Reply:
Yeah apparently we were not returning the total here but I can add it :)